glambie/data/timeseries.py

```diff
@@ -13,7 +13,8 @@
 class TimeseriesData():
     """Class to wrap the actual data contents of a Timeseries"""
     dates: np.ndarray
-    area: np.ndarray
+    area_reference: np.ndarray
```

Comment:
These names would ideally be better (unless they're also pinned by collaboration).

To give you an idea of how these can be misunderstood, my guess is that "area_reference" means the area over which the elevation at t=0 was calculated, and "area_observed" means the area over which the elevation at t=n was calculated.

Can you add some comments please explaining this?